src/components/MyPosts/MyPosts.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import "./MyPosts.css";
+import { getPostsByUserId } from "../../managers/PostManager";
+
+export const MyPosts = () => {
```

Comment:
Remember what we discussed about changes either to the front end or adding an elif in the do_POST method